### PR TITLE
Basics: clarify what's returned by metric name queries

### PR DESCRIPTION
**Pull request by tredondo**
Figured out how to sign off.

### CODE REVIEW

File: docs/querying/basics.md [hunk 1]
```diff
@@ -82,8 +82,8 @@ single sample value for each at a given timestamp (instant): in the simplest
 form, only a metric name is specified. This results in an instant vector
 containing elements for all time series that have this metric name.
 
-This example selects all time series that have the `http_requests_total` metric
-name:
+This example selects the sample at the current instant, for all time series
```
Comment (roidelapluie) [inline docs/querying/basics.md:85]: It is not at the current instance, which I understand as `now`. This depends on the parameters you pass to the API when doing the query. Also, it really selects all the samples. Wording it like "the sample" might make you think this returns just one sample.

cc @RichiH

Comment (RichiH) [reply docs/querying/basics.md:85]: It's also not the _instant_.

The line above it seems to already explain it correctly?
> This results in an instant vector containing elements for all time series that have this metric name.

Or am I missing something?

Comment (tredondo) [reply docs/querying/basics.md:85]: Pardon my noobness. I was running the queries while reading the docs for the first time, and what I noticed was that indeed one sample was returned per time series.

`curl https://play.victoriametrics.com/select/accounting/1/6a716b0f-38bc-4856-90ce-448fd713e3fe/prometheus/api/v1/query -d 'query=process_cpu_seconds_total'`

Comment (08d2) [reply docs/querying/basics.md:85]: > victoriametrics

Victoria Metrics != Prometheus

**Pull request closed without merging**